pkg/R/DataFrame.R

```diff
@@ -768,6 +798,20 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+setMethod("select",
+          signature(x = "DataFrame", col = "list"),
```

Comment:
Also, with the way the signature for `select` is written, will `do.call` actually work that way? The signature specifies one explicit `col` argument prior to `...`, so I was thinking that to pass a list to the current methods, you'd need something like this:
```R
arglist <- list("age", "name", "height")
select(df, arglist[[1]], arglist[2:3])
```
Does `do.call` collapse all the other arguments down into `...` for you?